### PR TITLE
Made a bot instance

### PR DESCRIPTION
**Pull request by Theflashiscool2**
I made the bot ready to set up and actually have a discord go instance

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -4,28 +4,51 @@ import (
 	// Import the Discordgo package, and other required packages.
```
Comment (z404) [inline main.go:4]: It looks like you missed out on import statements!

**Review by z404 — CHANGES_REQUESTED** [review]

**Comment by z404** [comment]
Hi @Theflashiscool2, Thank you for your contribution! I've requested a change, please take a look! I will also be assigning issue #5 to you, as this pr will be addressing the issue mentioned there. On future PRs, please comment on the issue you want to address so we can assign it to you.

Good work!

File: main.go (hunk 1 above)
Comment (Theflashiscool2) [reply main.go:4]: Wtf i had it when I was coding it should be fixed now tho

**Comment by Theflashiscool2** [comment]
Wtf i had the imports while coding.. that’s strange. Anyway it’s fixed now

File: main.go (hunk 1 above)
Comment (z404) [reply main.go:4]: I think you missed another import `fmt`

File: main.go [hunk 2]
```diff
@@ -2,30 +2,56 @@ package main
...
+func main() {
+	dg, err := discordgo.New("Bot " + token)
+	if err != nil {
+		panic(err)
+	}
+	u, err := dg.User("@me")
+	if err != nil {
+		panic(err)
+	}
+	botID = u.ID
```
Comment (z404) [inline main.go:26]: botID here has not been initialized. As it is used outside of this function as well, you might have to declare it globally. Please take a look!

**Review by z404 — CHANGES_REQUESTED** [review]

**Comment by Theflashiscool2** [comment]
Mobile coding 😵‍💫😵‍💫😵‍💫 I need goland on my pc

**Comment by Theflashiscool2** [comment]
I don’t think there’s any issue with the botID I used it like that for my first discord bot I made and it was working fine

**Comment by z404** [comment]
Correct me if I'm wrong but the definition that you have in your bot [here](https://github.com/Theflashiscool2/Discordgo-Bot/blob/main/main.go#L45) is missing in the file you've committed to this repository.

**Comment by Theflashiscool2** [comment]
You’re right 😵‍💫😵‍💫😵‍💫
I’m so stupid I’m sorry I didn’t have it on my other bot it’s probably broken you solved an error for my old bot 🥲

File: main.go (hunk 2 above)
Comment (Theflashiscool2) [reply main.go:28]: Im stupid 😀😀

**Review by z404 — APPROVED** [review]
LGTM! Good work :D

**Pull request merged**